sceptre/scipool/templates/sc-product-s3-private-enc.yaml

```diff
@@ -53,6 +53,10 @@ Resources:
           Info:
             LoadTemplateFromURL: !Sub '${RepoRootURL}/v1.1.10/s3/sc-s3-encrypted-ra.yaml'
           Name: 'v1.1.10'
+        - Description: 'Include option to restrict bucket downloads to same region'
```

Comment:
`Last updated at ${StackDatetime}` on v1.1.10 should only be on the latest version.  Would you mind removing that on v1.1.10 and addeding it to the end of this description?